src/main/java/banking/account/constant/AccountName.java

```diff
@@ -0,0 +1,6 @@
+package banking.account.constant;
+
+public enum AccountName {
```

Comment:
```suggestion
public enum AccountType {
```
해당 클래스는 어떤 상수를 담고 있는지 생각하고 그 역할에 맞게 이름을 작성하는 것이 중요합니다